pom.xml

```diff
@@ -357,11 +357,17 @@
             <artifactId>workflow-basic-steps</artifactId>
             <version>2.6</version>
         </dependency>
+        
+        <!--
+            workflow-scm-step >= 2.6 requires a Jenkins core >= 2.60.
+            Do not upgrade this dependency without a discussion on jenkinsci-ux@googlegroups.com
+        -->
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-scm-step</artifactId>
-            <version>2.6</version>
+            <version>2.5</version>
```

Comment:
@michaelneale @vivek this is becoming problematic https://github.com/jenkinsci/workflow-scm-step-plugin/pull/16#discussion_r125376152